test/extended/networking/multicast.go

```diff
@@ -28,6 +28,10 @@ var _ = Describe("[Area:Networking] multicast", func() {
 			oc := testexutil.NewCLI("multicast", testexutil.KubeConfigPath())
 			f := oc.KubeFramework()
 
+			BeforeEach(func() {
+				waitForNetNamespace(f.Namespace.Name)
```

Comment:
This first test only runs under ovs-subnet, which doesn't create NetNamespaces, so this will always fail